doc/rst/source/debug.rst

```diff
@@ -119,3 +121,60 @@ Applications).  Xcode may change as versions change; the images below is for Xco
    .. figure:: /_images/xcode-8.*
       :width: 100%
       :align: center
+
+Debug PyGMT in Xcode on macOS
+------------------------------
+
+**Note**: Tested with Xcode 12.0.  Install PyGMT following the official instructions (if you already have the dev version then you may just
+need to cd into your pygmt dir and call git pull):
+
+#. Add conda-forge channel::
+
+    conda config --prepend channels conda-forge
+    **Note**: The next step is different from the PyGMT official instructions, because we want to use the GMT dev version
+    conda create --name pygmt python=3.8 pip numpy pandas xarray netcdf4 packaging
+
+#. Activate the PyGMT environment::
+
+    conda activate pygmt
+
+#. Install Pygmt in editable/development mode::
+
+    cd pygmt
+    make install
```

Comment:
```suggestion
#. Add conda-forge channel and activate virtual environment (optional)::

    conda config --prepend channels conda-forge
    **Note**: The next step is different from the PyGMT official instructions, because we want to use the GMT dev version
    conda create --name pygmt python=3.8 pip numpy pandas xarray netcdf4 packaging

    # Activate the PyGMT environment
    conda activate pygmt

#. Install Pygmt in editable/development mode::

    cd pygmt
    pip install --editable .
```

Something like this?